### PR TITLE
Course 1 Module 1 notes

### PR DESCRIPTION
**Pull request by krauseannelize**
Added notes for Module 1 in course 1. Decided to close branch for module instead of course. Will continue branching for modules.

@skjoldar what do you think of the use of Mermaid diagrams?

### CODE REVIEW

File: 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+### Where are the gaps in our process?
+
+**Gap analysis** is a method for examining and evaluating how a process works currently in order to get to an improved future state. Businesses conduct gap analysis to improve a product or become more efficient, amongst others. The general approach to gap analysis is:
+
+- defining the current state,
+- defining where where you want to be or desired state,
+- analyzing the gap between the current and desired state to understand the difference, and
+- develop a plan to bridge the gap and determine which steps to take.
+
+### What did we not consider this before?
```
Comment (skjoldar) [inline 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md:182]: Fix grammar

Comment (skjoldar) [reply 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md:182]: Line 182 for the above

**Review by skjoldar — CHANGES_REQUESTED** [review]
Check line 182 in 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md

File: 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md (hunk 1 above)
Comment (krauseannelize) [reply 1-Foundations-Data-Data-Everywhere/1-Introducing-data-analytics.md]: Done - nice spot! First time doing a change request

**Comment by skjoldar** [comment]
Nice. merge when ready

**Pull request merged**